src/NuGet.Core/NuGet.Protocol/HttpSource/HttpHandlerResourceV3Provider.cs

```diff
@@ -48,6 +48,13 @@ private static HttpHandlerResourceV3 CreateResource(PackageSource packageSource)
                 AutomaticDecompression = (DecompressionMethods.GZip | DecompressionMethods.Deflate)
             };
 
+#if IS_DESKTOP
```

Comment:
If people set the MaxHTTPRequestPerSource explicitly we should *always* respect that.

Doesn't this break that in dotnet.exe?

Reply:
In dotnet.exe we don't need to configure `HttpClientHandler.MaxConnectionsPerServer` because it is default `Int32.MaxValue`.

If customers have configured `MaxHttpRequestsPerSource` then throttling requests is controlled by following code in both `.NET` and `.NET Framework` code paths. we don't change the value to `64` (which is new default). 

https://github.com/NuGet/NuGet.Client/blob/a56c42dbdd901c0c611eece400ce8565fffdfdf7/src/NuGet.Core/NuGet.Protocol/HttpSource/HttpSourceResourceProvider.cs#L46-L49